sdk/react/ooyalaSkinPanelRenderer.js

```diff
@@ -112,6 +112,7 @@ OoyalaSkinPanelRenderer.prototype.renderVideoView = function() {
       captionStyles={{
         textSize: this.skin.state.ccTextSize,
         textColor: this.skin.state.ccTextColor,
+        fontName: this.skin.state.ccFontName,
```

Comment:
no. we are not using this for android. we are using it only for ios and it is not crashing android.